migrations/versions/9a724381cde7_add_registry_instrument_count_view.py

```diff
@@ -20,7 +20,11 @@ def upgrade():
         """CREATE VIEW ras_ci.registry_instrument_count as SELECT exercise_id, count(*) as count
         FROM ras_ci.registry_instrument GROUP BY exercise_id; """
     )
+    op.create_index(
+        "ix_registry_instrument_exercise_id", "registry_instrument", ["exercise_id"], unique=False, schema="ras_ci"
+    )
```

Comment:
We wouldn't normally edit previous migrations, but this is the only way to all us to deal with dev, preprod and prod, all of which are at different revisions. Unlike Java Liquibase, these Alembic migrations do not consider a hash of the migrations files and therefore it's easier to move these statements between migration files. This is a last resort to ensure consistency throughout our envs.